docs/archive-data.md

```diff
@@ -0,0 +1,314 @@
+
+Get started with Google Cloud storage here: https://cloud.google.com/storage
+Get started with S3 storage here: https://aws.amazon.com/s3
+
+You will need the URI of the storage location where you want the data to be archived.
+
+If you want to create your own archiver, review the [Custom archiver](#custom-archiver) section below. Contributions are welcome.
+
+### Archival configuration
+
+First familiarize yourself with the configuration in [`config/development.yaml`](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81).
+
+| Config |  Description |
+|--------|--------------|
+| `archival.history.state`                       | Acceptable values are `"enabled"`,`"disabled"`. This must be set to `"enabled"` to use the Archival feature with any Namespace in the cluster. |
+| `archival.history.enableRead`                  | Acceptable values are `true`, `false`. This must be set to `true` to read from the archived Event History. |
+| `archival.history.provider`                    | Sub `provider` configs are `filestore`, `gstorage`, `s3`, or `your_custom_provider`. The default config specifies `filestore`. |
+| `archival.history.provider.filestore.fileMode` | This specifies the file permissions of the archived files. We recommend using the default value of `"0666"` to avoid read/write issues. |
+| `archival.history.provider.filestore.dirMode`  |  This specifies the directory permissions of the archive directory. we recommend using the default value of `"0766"` to avoid read/write issues. |
+| `namespaceDefaults.archival.history.state`     | Acceptable values are `"enabled"`, `"disabled"`. This sets the default state of the Archival feature whenever a new Namespace is created without specifying the Archival state. |
+| `namespaceDefaults.archival.history.URI`       | The value must be a URI of the file store location and match a schema that correlates to a provider. |
+
+Consider the following example:
+
+```yaml
+# Cluster level Archival config
+archival:
+  # Event History configuration
+  history:
+    # State at the cluster level
+    state: "enabled"
+    # Read permissions
+    enableRead: true
+    # Available providers
+    provider:
+      # Local filestore provider  
+      filestore:
+        # File permissions
+        fileMode: "0666"
+        # Directory permissions
+        dirMode: "0766"
+      # Google Cloud provider  
+      gstorage:
+        # Google Cloud credentials file
+        credentialsPath: "/tmp/gcloud/keyfile.json"
+
+# Default values for a Namespace if none are provided at creation
+namespaceDefaults:
+  # Archival defaults
+  archival:
+    # Event History defaults
+    history:
+      # Default state
+      state: "enabled"
+      # Default URI
+      URI: "file:///tmp/temporal_archival/development"
+```
+
+In the configuration above we see that:
```

Comment:
I'm not sure this section provides value but it definitely doesn't with how it's located on the page currently.